app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module DashboardHelper
+  def complaints_created
```

Comment:
Calling this method I'd expect a collection, not a number
I think a more descriptive name would be `number_of_created_complaints`
But because I see it's scoped by a user, maybe `number_of_created_complaints_by_user` IDK hehe